src/main/fc/fc_core.c

```diff
@@ -308,23 +316,19 @@ void tryArm(void)
             return;
         }
 #ifdef USE_DSHOT
-        if (isMotorProtocolDshot() && isModeActivationConditionPresent(BOXFLIPOVERAFTERCRASH)) {
+        if (isMotorProtocolDshot() && isModeActivationConditionPresent(BOXFLIPOVERAFTERCRASH) && !feature(FEATURE_3D)) {
```

Comment:
Not sure about moving the check for `FEATURE_3D` to here - I thought we'd implemented support for spinning individual motors with 3d enabled as well, so `flipOverAfterCrashMode` should probably be enabled here.

Reply:
Ok I wasn't aware that was working, last time I tried it (awhile back) it was messed up.